source/reference/verification/embedded.txt

```diff
@@ -20,7 +20,9 @@ Verify with Embedded Verifier
 
 ``mongosync`` includes an embedded verifier to perform a series
 of checks on the destination cluster to verify the sync of
-supported collections. 
+supported collections. Starting in version 1.9, ``mongosync`` enables
+the verifier by default on replica set clusters. In version 1.10, 
+``mongosync`` enables the verifier by default on sharded clusters.
```

Comment:
I think it may be better to omit the 1.9 and instead say in a separate line something like: "mongosync enables the embedded verifier by default"
and then under it new in 1.10 

Similar to how it is done below on this page I think where it says ".. versionadded:: 1.9"